astropy/utils/console.py

```diff
@@ -97,28 +88,6 @@ def ipyio(cls):
                 cls._ipyio = io
         return cls._ipyio
 
-    @classmethod
-    def get_stream(cls, stream):
-        return getattr(cls.ipyio, stream)
-
-
-def _get_stdout(stderr=False):
-    """
-    This utility function contains the logic to determine what streams to use
-    by default for standard out/err.
-
-    Typically this will just return `sys.stdout`, but it contains additional
-    logic for use in IPython on Windows to determine the correct stream to use
```

Comment:
tl;dr -- Is "IPython on Windows" also not a problem anymore? Not obvious if this is tied to IPython 4 or not.

Reply:
Apparently you didn't think this was a problem when you removed all the Ipython-related code from this function in 5ad0a5c83e6dafdc6dbdbef8c52137383b3abccf.

Reply:
I cannot tell you what I was thinking anymore over 3 years ago in #10942 . Looks like I was just trying get rid of deprecation warning but this was not affected. Hmm.